pkgs/development/tools/build-managers/alire/default.nix

```
@@ -0,0 +1,35 @@
+{ lib, stdenv, fetchFromGitHub, gprbuild, gnat }:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "alire";
+  version = "1.2.2";
+
+  src = fetchFromGitHub {
+    owner = "alire-project";
+    repo = "alire";
+    rev = "v${finalAttrs.version}";
+    hash = "sha256-rwNiSXOIIQR1I8wwp1ROVOfEChT6SCa5c6XnTRqekDc=";
+
+    fetchSubmodules = true;
+  };
+
+  nativeBuildInputs = [ gprbuild gnat ];
+
+  # on HEAD (roughly 2c4e5a3), alire provides a dev/build.sh script. for now,
+  # just use gprbuild.
+  buildPhase = ''
+    gprbuild -j0 -P alr_env
```

```suggestion
    gprbuild -j$NIX_BUILD_CORES -P alr_env
```
We can use this variable here.